### ISSUE TITLE
bug in debian.plugin.zsh

### ISSUE THREAD

**Issue by ealdekoa** [robbyrussell/oh-my-zsh#4804]
the lines 10 and 19 have which -p but -p is wrong option, the correct option is -a (which -a)
https://github.com/robbyrussell/oh-my-zsh/blob/master/plugins/debian/debian.plugin.zsh

**Comment by ealdekoa** [comment]
i dont now why but with zsh the option -p works, wich dont have the option...

**Comment by mcornella** [comment]
[`which` is a zsh builtin](http://zsh.sourceforge.net/Doc/Release/Shell-Builtin-Commands.html#index-which), which is a shortcut for `whence -c`. The [`whence` builtin](http://zsh.sourceforge.net/Doc/Release/Shell-Builtin-Commands.html#index-whence), as you can see in the zsh guide, does have a `-p` argument. This makes it behave such that it only looks for commands, without showing aliases or functions that might be named the same way. 

**TL;DR** that's correct code (though it could be improved by using `$commands`, but that's another topic). If you get any errors please post them.
